### PR TITLE
Adds several test suites

### PR DESCRIPTION
**Pull request by Ralt**
I've written the tests for: commands, events, jumps.

Missing test suites (but the boilerplate is there): keys, settings, user.

I've also removed `command-match` because it wasn't used anywhere.

### CODE REVIEW

**Comment by Ralt** [comment]
Tests fail because of an error in `sudo apt-get install libgtk-3.0 libwebkit2gtk-4.0-dev`

File: test/commands.lisp [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      (lispkit::command-p "test-command")
+    (5am:is-true (eq presentp t))))
+
+(5am:test test-run-named-command
+  (lispkit:defcommand test-command (browser)
+    "Test command"
+    (eq browser "fixture"))
+  (5am:is-true (eq (lispkit::run-named-command "test-command" "fixture") t)))
+
+(5am:test test-load-rc-file
```
Comment (jorams) [inline test/commands.lisp]: This test, and by extension the one below, fail when no configuration file is available.

File: test/commands.lisp [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  (lispkit:defcommand test-command ()
+    "Test command")
+  (multiple-value-bind (value presentp)
+      (lispkit::command-p "test-command")
+    (5am:is-true (eq presentp t))))
+
+(5am:test test-run-named-command
+  (lispkit:defcommand test-command (browser)
+    "Test command"
+    (eq browser "fixture"))
```
Comment (jorams) [inline test/commands.lisp]: I don't believe the CL standard requires implementations to translate two instances of the same literal string into  the same object, so this test can fail without there actually being a problem. `EQ` is not appropriate for strings.

File: test/commands.lisp (hunk 1 above)
Comment (Ralt) [inline test/commands.lisp]: Oh, the joys of testing locally... Thanks!

File: test/commands.lisp (hunk 2 above)
Comment (Ralt) [inline test/commands.lisp]: Duh. I actually correctly use `string=` in other places, this one slipped my mind. Thanks!

**Comment by Ralt** [comment]
Last commit says it all on how to fix travis:

- Uses stable repo instead of vivid
- Uses specific version of cl-webkit, because new versions rely on libwebkit2gtk-4.0, while ubuntu utopic only has 3.0.

@AeroNotix I'll let you merge.

**Comment by AeroNotix** [comment]
Excellent stuff mate! :+1:

**Pull request merged**